src/components/Popups/PopupHighlight.scss

```diff
@@ -0,0 +1,41 @@
+@import '~box-ui-elements/es/styles/variables';
+@import './PopupArrow';
+
+.ba {
+    $popup-arrow-size: 4px;
+    $popup-content-bg: $bdl-gray;
+    $popup-icon-margin: 8px;
```

Comment:
Probably doesn't need to be a variable if it's only used once. Same for $popup-arrow-size.